tribe-ext-relabeler/src/Tribe/Settings_Helper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+defined( 'WPINC' ) || die;
```

Comment:
In the above file, instead of this one-liner you use:
```
// Do not load directly.
if ( ! defined( 'ABSPATH' ) ) {
	die( '-1' );
}
```

If there's not a specific reason _why_ you'd look for `WPINC` here instead of `ABSPATH`, I'd recommend using the same check in both files uniformly.